pkg/client/interfaces/v1/options.go

```diff
@@ -146,22 +146,29 @@ func NewSettingsConfigurationOptions() *SettingsConfigurationOptions {
 			},
 		},
 		Agent: Agent{
+			Language: "en",
 			Listen: Listen{
-				Model: "nova-2",
+				Provider: ListenProvider{
+					Type:  "deepgram",
+					Model: "nova-3",
+				},
 			},
 			Think: Think{
-				Provider: Provider{
-					Type: "", // Required to be set
+				Provider: ThinkProvider{
+					Type:  "open_ai",
```

Comment:
I'm not sure if we want these think options hard coded here. Doesn't this mean if they don't change the options they'll automatically use open_ai?  in the API spec these are just enums:  

`enum: ["open_ai", "anthropic", "x_ai"]`

Reply:
Let's align on this tomorrow.

If we do not include a default option, and the user does not set one, they'll get this error:

![image](https://github.com/user-attachments/assets/bc8e9560-ebe1-47d5-858d-00adf6e045ad)

IMHO, 1:1 parity with the API is less desirable than streamlining the user's path from "install sdk" to "run it".